static/js/impala/ajaxcache.js

```diff
@@ -92,6 +92,9 @@ $.ajaxCache = function(o) {
             }
         }
     }
+    $("#live-announcements").html(gettext(
```

Comment:
Our coding style generally prefers using single quotes over double-quotes in JS.